packages/@aws-cdk/aws-cognito/lib/user-pool-client.ts

```diff
@@ -297,15 +310,14 @@ export class UserPoolClient extends Resource implements IUserPoolClient {
   }
 
   private configureOAuthScopes(oAuth?: OAuthSettings): string[] | undefined {
```

Comment:
```suggestion
  private configureOAuthScopes(oAuth?: OAuthSettings): string[] {
```

Reply:
But there's a code path that does return `undefined`.

Reply:
Scratch that.